spec/features/user_stories_spec.rb

```diff
@@ -0,0 +1,42 @@
+describe 'User Stories' do
+
+  describe 'Takeaway' do
+    # As a customer
+    # So that I can check if I want to order something
+    # I would like to see a list of dishes with prices
+    it 'should display a list of dishes with prices' do
+      takeaway = Takeaway.new
+      expect(takeaway.list).not_to be_empty
+    end
+
+    # As a customer
+    # So that I can order the meal I want
+    # I would like to be able to select some number of several available dishes
+    it 'should allow user to select some number of several dishes' do
+      takeaway = Takeaway.new
+      takeaway.choose("Dish 1")
+      expect(takeaway.order).not_to be_empty
+    end
+
+    # As a customer
+    # So that I can verify that my order is correct
+    # I would like to check that the total I have been given matches the sum of
+    # the various dishes in my order
+    it "the customer's total should match the sum of dishes in the order" do
+      takeaway = Takeaway.new
+      takeaway.choose("Dish 1", 3)
+      takeaway.calculate_quantities(takeaway.order)
+      takeaway.calculate_cost(takeaway.order)
+      expect{ takeaway.place_order(takeaway.order,takeaway.quantity,takeaway.total) }.not_to raise_error
```

Comment:
Metrics/LineLength: Line is too long. [104/80] (https://github.com/bbatsov/ruby-style-guide#80-character-limits)